suse_openstack_cloud

```diff
@@ -390,3 +391,18 @@ if [ -e /opt/dell/crowbar_framework/.crowbar-installed-ok ]; then
   find_and_plog_files_0 $chef_export_dir -type f
   rm -rf $chef_export_dir
 fi
+
+###############################################################
+section_header "RabbitMQ broker"
+
+if [ -e /usr/sbin/rabbitmqctl ]; then
+    rabbitmq_tmpdir=$(mktemp -d)
+    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    source /etc/rabbitmq/rabbitmq-env.conf
+    pushd `pwd`
+    cd /var/lib/rabbitmq
+    su rabbitmq -s /bin/sh -c "/usr/lib64/rabbitmq/bin/rabbitmqctl report" > ${rabbitmq_report_log}
+    popd
+    find_and_plog_files_0 ${rabbitmq_tmpdir} -type f
```

Comment:
So this was maybe [my fault for not thinking about it in the right way back in February](https://github.com/SUSE-Cloud/supportutils-plugin-suse-openstack-cloud/pull/33#discussion_r24925890), or maybe we changed our approach slightly since I wrote that.  But either way, since then we have this plugin writing files to

    /var/log/crowbar/batch-export.log
    /var/log/crowbar/chef-export/

then running `plog_files` on them before deleting them.  So it makes sense for this new rabbitmqctl capture does the same thing, even if I said not to do that in February.  Hindsight is a valuable thing :-)